android/src/main/java/com/rnlib/adyen/AdyenPaymentModule.kt

```diff
@@ -564,9 +565,13 @@ class AdyenPaymentModule(private var reactContext : ReactApplicationContext) : R
     @Suppress("UNUSED_PARAMETER")
     private fun parseActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
         Log.d(TAG, "parseActivityResult")
-        if (requestCode == DropIn.DROP_IN_REQUEST_CODE && resultCode == Activity.RESULT_CANCELED && data != null) {
-//            sendFailure("ERROR_CANCELLED","Transaction Cancelled")
-            Log.d(TAG, "DropIn CANCELED")
+        if (resultCode == Activity.RESULT_CANCELED) {
+            if (requestCode == AdyenComponent.DROP_IN_REQUEST_CODE && data != null) {
+                sendFailure("ERROR_CANCELLED", "Transaction Cancelled")
+                Log.d(TAG, "DropIn CANCELED")
```

Comment:
Would there be value in logging this when a user cancels Google Pay too?

Reply:
No, that part is not touched when user cancels Google Pay. Maybe in the future if we want to track other payment options it would have a value